contracts/Balancer/Balancer.sol

```diff
@@ -217,12 +217,23 @@ contract Balancer is NonblockingLzApp, Owned {
         IStabilizer(stabilizer).setLoanLimit(loanLimit);
     }
 
+    function resetCalls() external {
+        address[] memory minterAddresses = sweep.getMinters();
+        uint256 len = minterAddresses.length;
+
+        for (uint256 i = 0; i < len; ) {
+            cancelCall(minterAddresses[i]);

```

Comment:
IStabilizer stabilizer = IStabilizer(stabilizers[i]);
uint256 callAmount = stabilizer.callAmount;
if(callAmount == 0) cancelCall(minterAddresses[i]);

Reply:
I see. Good! the condition should be callAmount > 0 though.